packages/web/src/pages/sign-on-page/SignOnPage.tsx

```diff
@@ -0,0 +1,162 @@
+import { useEffect } from 'react'
+
+import {
+  Box,
+  Flex,
+  IconCloseAlt,
+  Paper,
+  Text,
+  TextLink,
+  useTheme
+} from '@audius/harmony'
+import { Link, Route, Switch } from 'react-router-dom'
+import { useToggle } from 'react-use'
+
+import djBackground from 'assets/img/2-DJ-4-3.jpg'
+import djPortrait from 'assets/img/DJportrait.jpg'
+import BackgroundWaves from 'components/background-animations/BackgroundWaves'
+import { useMedia } from 'hooks/useMedia'
+import { SignInPage } from 'pages/sign-in-page/SignInPage'
+import { AudiusValues } from 'pages/sign-on-page/AudiusValues'
+import SignUpPage from 'pages/sign-up-page'
+import { SIGN_IN_PAGE, SIGN_UP_PAGE, TRENDING_PAGE } from 'utils/route'
+
+const messages = {
+  newToAudius: 'New to Audius?',
+  createAccount: 'Create an Account'
+}
+
+export const SignOnPage = () => {
+  const { isMobile } = useMedia()
+  const [isExpanded] = useToggle(false)
```

Comment:
is this a placeholder for a future control? If we don't even use the toggle callback, this will always be false

Reply:
yeah you found it! this will get called when we move to password page on mobile, and when we move to create handle in desktop.